### DISCUSSION TITLE
Numpy 3D visualization

### DISCUSSION THREAD

**Discussion by alexeytopolnitskiy** [pyvista/pyvista#1408] [Q&A] [answered]
Hello!

I have a numpy 3d array which is a representation of a porous media. There are 2 values of voxels in the model: 255 is for solid part and 0 is for pores. I want to create an interactive 3D plot to analyze the model but I am still struggling to find an appropriate way. Is it possible to do it with PyVista?

**Comment by akaszynski** [comment, marked as answer]
Hello,

You'll probably want to create a structured grid out of this and then use a threshold filter.  See: https://docs.pyvista.org/core/point-grids.html
